src/js/dataMapping/awards/additionalDetailsIdv.js

```diff
@@ -66,15 +66,15 @@ const additionalDetails = (awardData) => {
             Recipient: {
                 type: 'link',
                 data: {
-                    path: recipient.internalId ? `/#/recipient/${recipient.internalId}` : null,
+                    path: recipient.internalId ? `/#/recipient/${recipient.internalId}/latest` : null,
                     title: recipient._name
                 }
             },
             DUNS: recipient.duns || '',
             'Parent Recipient': {
                 type: 'link',
                 data: {
-                    path: recipient.parentInternalId ? `/#/recipient/${recipient.parentInternalId}` : null,
+                    path: recipient.parentInternalId ? `/#/recipient/${recipient.parentInternalId}/latest` : null,
```

Comment:
Why do we have to set all these to latest? Doesnt the page already go to latest if no fy?